src/i18n/langResources/lang.de.ts

```diff
@@ -20,18 +20,18 @@ export const langDe: ILangResource = {
   reload: 'Neu starten',
   autoReload: 'Automatisch neu starten',
   disableDetect: 'Projekterkennung deaktivieren',
-  iconCustomization: 'Das Icon-Manifest wird regeneriert. ',
-  iconRestore: 'Das Icon-Manifest wird zurückgesetzt. ',
-  ngPresetEnabled: 'Angular Icons wird aktiviert. ',
-  ngPresetDisabled: 'Angular Icons wird deaktiviert. ',
-  jsOfficialPresetEnabled: 'Offizielles JS Icon wird aktiviert. ',
-  jsOfficialPresetDisabled: 'Offizielles JS Icon wird deaktiviert. ',
-  tsOfficialPresetEnabled: 'Offizielles TS Icon wird aktiviert. ',
-  tsOfficialPresetDisabled: 'Offizielles TS Icon wird deaktiviert. ',
-  jsonOfficialPresetEnabled: 'Offizielles JSON Icon wird aktiviert. ',
-  jsonOfficialPresetDisabled: 'Offizielles JSON Icon wird deaktiviert. ',
-  hideFoldersPresetEnabled: 'Ordner Icons werden aktiviert. ',
-  hideFoldersPresetDisabled: 'Ordner Icons werden deaktiviert. ',
+  iconCustomization: 'Das Icon-Manifest wird regeneriert.',
+  iconRestore: 'Das Icon-Manifest wird zurückgesetzt.',
+  ngPresetEnabled: 'Angular Icons wird aktiviert.',
+  ngPresetDisabled: 'Angular Icons wird deaktiviert.',
+  jsOfficialPresetEnabled: 'Offizielles JS Icon wird aktiviert.',
+  jsOfficialPresetDisabled: 'Offizielles JS Icon wird deaktiviert.',
+  tsOfficialPresetEnabled: 'Offizielles TS Icon wird aktiviert.',
+  tsOfficialPresetDisabled: 'Offizielles TS Icon wird deaktiviert.',
+  jsonOfficialPresetEnabled: 'Offizielles JSON Icon wird aktiviert.',
+  jsonOfficialPresetDisabled: 'Offizielles JSON Icon wird deaktiviert.',
+  hideFoldersPresetEnabled: 'Ordner Icons werden aktiviert.',
```

Comment:
yes, we are. I'm submitting the German translation in the day, but I have to learn for school.